tests/test_jmcomic/test_jm_async_client.py

```
@@ -0,0 +1,264 @@
+"""
+Async Client API 对称性测试 —— 对标 test_jm_client.py
+
+每个测试同时调用 sync API client 和 async API client，diff 返回值。
+若 sync/async 行为不一致，测试 FAIL 并报告差异。
+"""
+
+from test_jmcomic import *
```

Ruff (0.15.15)

[error] `from test_jmcomic import *` used; unable to detect undefined names

(F403)